packages/ui-kit/src/lib/dimensions/index.ts

```diff
@@ -13,6 +13,7 @@ export const dimensions = {
   xl_6: '2.5rem', //40px
   base_3: '3rem', //48px
   xl_10: '3.125rem', //50px
+  wide_laptop_width: '85rem', //1360px
```

Comment:
Usually, the next grade is `1440px`, but if that could break the layour we can keep it as it is